twitchpopups.htm

```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
                   hotseatison = true;
                   $("#popupbox").show();
                   hotseatuser = commandName.substr(10).toLowerCase();
-                  $("#popupbox").css({"background-color":"#aa1100"}); 
-                  $("#popuptext").text("🔥 " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT 🔥");
+                  $("#popupbox").css({"background-color":hotseatBg}); 
+                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);

```

Comment:
better to use template literals here
`$("#popuptext").text(`${hotseatEmoji} ${hotseatuser.toUpperCase()} IS IN THE HOTSEAT ${hotseatEmoji}`);`

Reply:
This is a joyful and wondrous suggestion!

Reply:
Preferable, rather than better I think

Reply:
how's it going fellas

Reply:
Beyond the scope of my changes. I was setting up configuration variables, not refactoring.